lib/connections/service.go

```diff
@@ -401,7 +401,7 @@ func (s *Service) connect() {
 
 				conn, err := dialer.Dial(deviceID, uri)
 				if err != nil {
-					l.Debugln("dial failed", deviceCfg.DeviceID, uri, err)
+					l.Infof("%v for %v at %v: %v", dialerFactory, deviceCfg.DeviceID, uri, err)
```

Comment:
This might flood, as we might have old addresses, so better this stays debug.

Reply:
ok